### ISSUE TITLE
Wrong cursor for disabled state for dropdown component

### ISSUE THREAD

**Issue by theiliad** [IBM/carbon-components-angular#495]
**Current behavior:**
![image](https://user-images.githubusercontent.com/14989804/56066293-5dd6de00-5d45-11e9-8d35-e66785ce5939.png)
(Hover state appears and **you get the hand cursor**)

**Carbon React:**
![image](https://user-images.githubusercontent.com/14989804/56066340-7941e900-5d45-11e9-9ffa-1d901466935d.png)
(No hover, and cursor is `not-allowed`)

**Comment by cal-smith** [comment]
`.bx--dropdown--disabled` is missing on the `.bx--list-box` div
